conf/stage.env

```diff
@@ -6,4 +6,4 @@
 # will be used to prefix the CloudFormation stack names as well as the s3
 # bucket that contains revisions of the stacker templates. This is the only
 # required environment variable.
-namespace:
+namespace: stacker_blueprints
```

Comment:
If this is left blank, what error happens if you run this?

Reply:
I got this error

>stacker.exceptions.InvalidConfig: {'namespace': ConversionError([ErrorMessage(u'This field is required.', None)])}